livekit-ffi/src/server/room.rs

```diff
@@ -465,6 +465,28 @@ impl RoomInner {
         server.watch_panic(handle);
         proto::UpdateLocalNameResponse { async_id }
     }
+
+    pub fn update_local_attributes(
+        self: &Arc<Self>,
+        server: &'static FfiServer,
+        update_local_attributes: proto::UpdateLocalAttributesRequest,
+    ) -> proto::UpdateLocalAttributesResponse {
+        let async_id = server.next_id();
+        let inner = self.clone();
+        let handle = server.async_runtime.spawn(async move {
+            let _ = inner
+                .room
+                .local_participant()
+                .update_attributes(update_local_attributes.attributes)
+                .await;
+
+            let _ = server.send_event(proto::ffi_event::Message::UpdateLocalAttributes(
+                proto::UpdateLocalAttributesCallback { async_id },
+            ));
+        });
+        server.watch_panic(handle);
+        proto::UpdateLocalAttributesResponse { async_id }
```

Comment:
more of a question/note about request/response semantics in the FFI layer

we are planning to allow set_attributes/set_metadata to be able to return failure & error string. This could be due to the participant not having permission to update that field. How do we reflect these errors on the Rust/FFI side?

Reply:
We already expose the error, but this PR ignores it above